app/controllers/concerns/authenticatable.rb

```diff
@@ -0,0 +1,51 @@
+module Authenticatable
```

Comment:
📝

This is a general AuthN helper but mostly deals with SSO for now since Devise internally handles the rest. 

The `skip_authentication` is the only general method right now.